config/webhook/service.yaml

```diff
@@ -4,6 +4,8 @@ kind: Service
 metadata:
   name: webhook-service
   namespace: system
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: webhook-server-cert
```

Comment:
This secret is currently generated by the certmanager. Where do you generate it in openshift deployment?

Reply:
that is an OpenShift feature that will generate that secret.
https://docs.openshift.com/container-platform/4.6/security/certificates/service-serving-certificate.html#add-service-certificate_service-serving-certificate